### PR TITLE
butter is slippery

### PR DESCRIPTION
**Pull request by IProduceWidgets**
## Why / Balance

You would expect it to be if you stepped on it, right?

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: butter is slippery

### CODE REVIEW

**Comment by IProduceWidgets** [comment]
test fail unrelated

**Comment by ps3moira** [comment]
I suggest making it slippery but maybe a RNG for the chance to slip on it? Feel like it shouldn't be 100% of the time considering you can make butter.

**Comment by deltanedas** [comment]
you can make soap banana peels etc too

slipping should be based on some (probably random) factor of where your foot is, stepping somewhere near butter shouldnt slip you but landing on the butter should

**Comment by ps3moira** [comment]
> you can make soap banana peels etc too
> 
> 
> 
> slipping should be based on some (probably random) factor of where your foot is, stepping somewhere near butter shouldnt slip you but landing on the butter should

I imagine there will be a future rotting mechanic for fruits and vegetables, so that won't be much an issue. You can make a ton of butter if you have the power-cow set up though and butter doesn't rot at all

**Comment by regenpower** [comment]
imo slipping should not be random, I'd much rather see it nerfed by having the ability to pick someone up who slipped instead of making slipping RNG

File: Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml [hunk 1]
```diff
@@ -475,10 +476,35 @@
   components:
   - type: Sprite
     state: butter
+  - type: Slippery
+    paralyzeTime: 1.5
+    launchForwardsMultiplier: 1.2
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml:481]: You might want to adjust the launchForwardMultiplier to 1.5 so it is in accordance with the new standardised slip stats #27879

Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml:481]: Oh yea can do. I intended for it to be less good than soap and more of a kitchen drop hazard when soap was stronger, but I'm not so fussed about it. I'll just yoink the paralyzeTime and launchForwardMultiplier fields from being set.

**Pull request merged**